bundles/org.openhab.binding.harmonyhub/src/main/java/org/openhab/binding/harmonyhub/internal/discovery/HarmonyHubDiscoveryService.java

```diff
@@ -171,6 +171,7 @@ private void sendDiscoveryMessage(String discoverString) {
             // Broadcast the message over all the network interfaces
             Enumeration<NetworkInterface> interfaces = NetworkInterface.getNetworkInterfaces();
             while (interfaces.hasMoreElements()) {
+                @Nullable
```

Comment:
Is it necessary? 
You add nullable but then you do not check if the value is null.

Reply:
See https://github.com/openhab/openhab-addons/pull/12303#discussion_r810625052

Reply:
Ok, strictly the same thing.